eZ/Publish/Core/Repository/UserService.php

```diff
@@ -61,6 +61,9 @@
  */
 class UserService implements UserServiceInterface
 {
+    private const MISSING_CONTENT_TYPE_EZUSER_FIELD_TYPE_ERROR_MESSAGE =
+        'The provided Content Type does not contain the ezuser Field Type';
+
```

Comment:
Can we introduce dedicated exception class (extending ContentValidationException) for this case? e.g. MissingFieldException

Reply:
Added `MissingUserFieldTypeException` in 3a0c98b